### DISCUSSION TITLE
Can we develop for ios on windows with capacitor?

### DISCUSSION THREAD

**Discussion by ivanjeremic** [ionic-team/capacitor#5249] [General] [answered]
Can we develop for ios on windows with capacitor?

**Comment by gmbett** [comment]
I don't think that is possible, you need a mac to develop for iOS (to have Xcode)

**Comment by jcesarmobile** [comment, marked as answer]
You can do the code editing from windows, but you need some local or remote Mac to do the native build.
Some examples of remote Mac or build services are macincloud, macstadium or ionic appflow. You can use those to build your app and test on a real device if you have one.
